pkgs/by-name/sc/scipopt-zimpl/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  bison,
+  flex,
+  gmp,
+  zlib,
+}:
+
+stdenv.mkDerivation {
+  pname = "scipopt-zimpl";
+  version = "362";
+
+  src = fetchFromGitHub {
+    owner = "scipopt";
+    repo = "zimpl";
+    rev = "v362";
+    sha256 = "juqAwzqBArsFXmz7L7RQaE78EhQdP5P51wQFlCoo7/o=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    bison
+    flex
+  ];
+
+  buildInputs = [
+    gmp
+    zlib
+  ];
+
```

```suggestion
  doCheck = true;

  checkPhase = ''
    runHook preCheck
    pushd ../check
    ./check.sh ../build/bin/zimpl
    popd
    runHook postCheck
  '';
```
Test suite is a bit more tricky to run.